public/friendlycode/js/friendlycode.js

```diff
@@ -77,9 +81,25 @@ define(function(require) {
       editor.panes.preview.setViewLink(info.viewURL);
     });
 
+    // Bramble: Load the project Files into the fs
+    var initFs = function(callback) {
+      if(!makeDetails || !makeDetails.title) {
+        makeDetails = ProjectFiles.generateDefaultProject();
+        ProjectFiles.load(makeDetails, defaultContent, callback);
+        return;
+      }
+
+      ProjectFiles.load(makeDetails, callback);
+    };
+
+    FileSystemSync.init(makeDetails && makeDetails.title, {
+      createOrUpdate: options.appUrl + "/updateProjectFile",
+      del: options.appUrl + "/deleteProjectFile"
```

Comment:
I can't use `Path.join` because it normalizes the result...so I end up with something like:
`Path.join("http://localhost:3500/", "updateProjectFile") = "http://localhost:3500/localhost:3500/updateProjectFile"`